### PR TITLE
Fix flutter proxy

### PR DESCRIPTION
**Pull request by jascodes**
Possible fix for #136

### CODE REVIEW

**Review by leoafarias — CHANGES_REQUESTED** [review]
It seems that linemode does work on Windows but you have to disable the echoMode also. According to the docs here. 
https://api.flutter.dev/flutter/dart-io/Stdin/lineMode.html

File: packages/cli/lib/src/flutter_tools/flutter_helpers.dart [hunk 1]
```diff
@@ -43,8 +44,13 @@ bool isGlobalVersion(String version) {
 
 String getFlutterSdkExec(String version) {
   // If version not provided find it within a project
+
   if (version == null || version.isEmpty) {
-    return whichSync('flutter');
+    final execPath = whichSync('flutter');
```
Comment (leoafarias) [inline packages/cli/lib/src/flutter_tools/flutter_helpers.dart:49]: Should we check if execPath is null or empty before telling the user that a version was found?

**Comment by jascodes** [comment]
> It seems that linemode does work on Windows but you have to disable the echoMode also. According to the docs here.
> [api.flutter.dev/flutter/dart-io/Stdin/lineMode.html](https://api.flutter.dev/flutter/dart-io/Stdin/lineMode.html)

Feel free to that then...

File: packages/cli/lib/src/flutter_tools/flutter_helpers.dart (hunk 1 above)
Comment (jascodes) [reply packages/cli/lib/src/flutter_tools/flutter_helpers.dart:49]: Well, in previous PR I did check it in the command file itself. But the I didn't wanted to make too much changes to the way to structured the code. So now it as using flutter in path and if not found. it prints error 'flutter not found in path'.

I am cool with any way to like.

**Pull request closed without merging**